src/audio/kpb.c

```diff
@@ -1593,6 +1594,44 @@ static enum task_state kpb_draining_task(void *arg)
 	return SOF_TASK_STATE_COMPLETED;
 }
 
+void kpb_convert_24b_to_32b(const void *linear_source, int ioffset,
```

Comment:
Consider declaring this function as static if it is not going to be used outside the module.

Reply:
ack - if it's an internal API we need static.